docs/api-examples/controller.md

```diff
@@ -0,0 +1,6 @@
+<h1>Controller</h1>
+
+
+All requests are authenticated using [basic auth](https://en.wikipedia.org/wiki/Basic_access_authentication) where the user name is blank and the password is your controller key. You may alternatively use the `key` URL parameter with the controller key for event streams (this allows using them via JavaScript in the browser).
+
```

Comment:
This should include the `flynn -a controller env get CONTROLLER_KEY` command (I think that's the name, test it to be sure)